### PR TITLE
Add global ponyfill

### PR DESCRIPTION
**Pull request by TrySound**
Ref https://github.com/cssinjs/jss/issues/736

### CODE REVIEW

File: src/utils/global.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/* eslint-disable no-nested-ternary */
+export default (typeof global !== 'undefined'
```
[the remark this answers is not in the document]
Comment (kof) [reply src/utils/global.js]: isn't there a plugin that does exact same thing that webpack does?

**Comment by kof** [comment]
https://github.com/cssinjs/jss/pull/741

**Comment by oliviertassinari** [comment]
I have faced the same problem on Material-UI side. I have dug into the problem for some time. 

> I had this exact code some years ago, then webpack fixed it for us. It would be nice to have it fixed by the build tool, because we we need it in many other plugins.

Yes, webpack is taking care of it. I have found this first clue: https://github.com/rollup/rollup-plugin-commonjs/issues/98. Then, undeeded up with this solution: https://github.com/mui-org/material-ui/pull/11878/files.

**Comment by kof** [comment]
yep, already done that

**Comment by kof** [comment]
https://github.com/cssinjs/jss/pull/741

**Pull request closed without merging**